tests/pkg/fixture/fixture.go

```diff
@@ -190,6 +191,14 @@ func GetTidbCluster(ns, name, version string) *v1alpha1.TidbCluster {
 			},
 		},
 	}
+
+	random := rand.Intn(2)
+	if random != 0 && version == utilimage.PDMSImage {
+		// 50% random in pdms mode
+		tc = AddPDMSForTidbCluster(tc)
+	}
```

Comment:
do we need any message to indicate we're running in MS mode? as this is a random behavior, if no information about this, debug test cases may get harder.

Reply:
added.